RealmSwift/Sync.swift

```diff
@@ -1616,3 +1616,32 @@ extension List where Element == Permission {
         return RLMPermissionForRole(_rlmArray, role) as! Permission
     }
 }
+
+#if canImport(Combine)
+import Combine
+
+@available(OSX 10.15, watchOS 6.0, iOS 13.0, iOSApplicationExtension 13.0, OSXApplicationExtension 10.15, tvOS 13.0, macCatalyst 13.0, macCatalystApplicationExtension 13.0, *)
```

Comment:
is the `@available` statement required if the whole thing is wrapped in `#if canImport(Combine)`?

Reply:
Yes, it's all required– the macro is "compiled" differently than the extension is.